CONTRIBUTING.md

```diff
@@ -26,26 +26,156 @@
+|Product feature          |feature-upgrades        |Upgrades-related work.                                                                                                                                        |                                                                                              |
+|Product feature          |feature-monitoring      |Monitoring-related work.                                                                                                                                      |                                                                                              |
+|Product feature          |feature-osa             |OSA-related work.                                                                                                                                             |                                                                                              |
+
+## Pre-Backlog -> Backlog
+### Adding an issue to the backlog
+Click [here](https://github.com/rcbops/rpc-openstack/issues/new) to load the form used when creating a new issue in the backlog. The form is generated from [this template](.github/ISSUE_TEMPLATE.md) which includes a set of questions for the reporter and a checklist to be completed by the owner as the issue is worked. Currently Waffle does not support the use of an issue template and so issues should not be created directly in Waffle.
+
+## Backlog
+All issues that are are not approved to be worked are listed in Backlog, this includes new issues and those that have been triaged but not yet committed to.
+
+### Issue triage
+Issue triage is an ongoing process that should happen on a daily basis, the engineering team will triage new issues and monitor unprioritised issues so that new commitments are always targetting the most import work.
+
+Each issue is given a priority and that priority is used to help decide the order in which issues are worked. It is important to remember that there is a finite resource available to fix issues. The process of prioritisation is used to focus the available resources on the most important tasks identified.
+
+Issues are labelled with a priority as defined in the [labels table](#Labels).
+
+#### Expedited issues
+Expedited issues are allowed to exceed the WIP limit on any column. These issues are critical to fix and must be prioritised above all other work. They are identified with the label 'prio-expedited'. Due to the negative impact expedited issues have on WIP it is important they are used appropriately.
+
+#### Enhancements
+Issues labeled as 'type-enhancement-large' or 'type-enhancement-medium' filter into planning for major and minor releases and should never be moved to Approved.
```

Comment:
If these are never in approved, does that mean they will be directly assigned to team members and jump from backlog to doing?